apps/condo/domains/organization/integrations/sbbol/sync/syncBankAccounts.js

```diff
@@ -82,13 +102,13 @@ const syncBankAccounts = async (userId, organization) => {
 
     if (!fintechApi) return
 
-    logger.info('Checking, whether the user have ClientAccount items')
+    logger.info({ msg: 'Checking, whether the user have ClientAccount items' })
```

Comment:
It seems to be redundant because in case when no accounts are returned by SBBOL, we have a oog record for that